sdcm/cluster.py

```diff
@@ -2778,6 +2818,21 @@ def __init__(self, ec2_ami_id, ec2_subnet_id, ec2_security_group_ids,
         self.seed_nodes_private_ips = None
         self.version = '2.1'
 
+    def populate_db(self, node):
+        node.remoter.run("sudo rm -rf /var/lib/scylla/*")
+        node.remoter.run("sudo cp -a /root/* /var/lib/scylla/")
+        node.remoter.run("sudo chown -R scylla.scylla /var/lib/scylla")
+        node.remoter.run("sudo rm -rf /var/lib/scylla/commitlog/*")
+
+    def stop_db(self, node):
+        node.remoter.run("sudo systemctl stop scylla-server")
+        node.wait_db_down()
+
+    def start_db(self, node):
+        node.remoter.run("sudo systemctl start scylla-server")
+        import time
+        time.sleep(45)
```

Comment:
Why do we sleep 45 seconds here only for scylla?
I saw we have wait_db_up() after that.